src/database/models/request.js

```diff
@@ -11,5 +12,10 @@ module.exports = (sequelize, DataTypes) => {
     status: DataTypes.STRING,
     accommodation: DataTypes.BOOLEAN
   });
+  request.associate = (models) => {
+    request.belongsTo(models.request, {
+      foreignKey: 'userId'
+    });
+  };
```

Comment:
@descholar-ceo Why are you associating request model to itself?

Reply:
It was a mistake, I am going to take care of it!

Reply:
I implemented this comment too